config/webpack.dev.config.js

```diff
@@ -7,28 +7,10 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const apiEndpoints = require('../src/data/api/endpoints.js');
 const commonConfig = require('./webpack.common.config.js');
 
-let targetUrl = 'localhost';
-if (!process.env.RUNNING_ON_LINUX) {
-  targetUrl = `docker.for.mac.${targetUrl}`;
-}
-targetUrl = `http://${targetUrl}:18010`;
+const targetUrl = 'http://edx.devstack.studio:18010';
 
 module.exports = Merge.smart(commonConfig, {
   devtool: 'cheap-module-eval-source-map',
-  entry: {
-    assets: [
-      // enable react's custom hot dev client so we get errors reported
-      // in the browser
-      require.resolve('react-dev-utils/webpackHotDevClient'),
```

Comment:
The stock webpack-dev-server can do the error overlay too, so I removed this in favor of a simpler webpack config and less dependencies.